attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/AttributeMetadataValidator.java

```diff
@@ -57,6 +59,14 @@ public static void validate(AttributeCreateRequest attributeCreateRequest) {
     }
   }
 
+  public static void validateDeletionFilter(final AttributeMetadataFilter attributeMetadataFilter) {
+    if (!attributeMetadataFilter.getCustom()) {
```

Comment:
This would mean all places that currently call delete would fail validation, since false is the ddefault value. We'd need to do a presence check (`hasCustom() && !getCustom()`) (and would need to make sure we're appropriately defaulting to true in the delete case, false in the query case)

Reply:
As covered by the new integration tests.

Reply:
> This is not required, right? Because, in the query case, we have 3 different meaning

Yes, sorry - my mixup. I meant defaulting to not treat it like false, but it shouldn't be true - it's as you described.